docs/sources/features/datasources/x-ray.md

```diff
@@ -0,0 +1,134 @@
+| _Default_                  | Default data source means that it will be pre-selected for new panels.                                                   |
+| _Default Region_           | Used in query editor to set region (can be changed on per query basis)                                                   |                                                                    
+| _Auth Provider_            | Specify the provider to get credentials.                                                                                 |
+| _Credentials_ profile name | Specify the name of the profile to use (if you use `~/.aws/credentials` file), leave blank for default.                  |
+| _Assume Role Arn_          | Specify the ARN of the role to assume                                                                                    |
+| _External ID_              | If you are assuming a role in another account, that has been created with an external ID, specify the external ID here. |
+
+## Authentication
+
+### AWS credentials
+
+If Auth Provider is `Credentials file`, Grafana tries to get credentials in the following order.
+
+- Environment variables. (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`)
+- Hard-code credentials.
+- Shared credentials file.
+- IAM role for Amazon EC2.
+
+See the AWS documentation on [Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+
+### AWS credentials file
+
+Create a file at `~/.aws/credentials`. That is the `HOME` path for user running grafana-server.
+
+> NOTE: If you think you have the credentials file in the right place but it is still not working then you might try moving your .aws file to '/usr/share/grafana/' and make sure your credentials file has at most 0644 permissions.
+
+Example content:
+
+```bash
+[default]
+aws_access_key_id = <your access key>
+aws_secret_access_key = <your access key>
+region = us-west-2
+```
+
+## The Query Editor 
+
+The most important field in the editor is the query type. There are 4 query types Trace List(Traces in AWS), Trace Statistics, Trace Analytics(Analytics in AWS), Insights.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-query-editor.png" max-width="800px" class="docs-image--left" caption="X-Ray query editor" >}}
+
+### Trace List
+
+The trace list is pretty much the same as in AWS. Clicking on the trace id in the first column opens the trace on the right side. Notice the query field in the editor which works the same as in AWS. You can write queries, filter expressions or a trace id there.
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-trace-list.png" caption="X-Ray trace list in Grafana" >}}
+
+### Trace Statistics
+
+In trace statistics you can see a graph and a table showing information about error, fault, throttle, success and total count. You can use the columns field in the query editor to only see specified columns.
```

Comment:
Per the style guides, use the serial comma. Apply throughout.

Reply:
done